packages/backend/src/spsp/service.ts

```diff
@@ -12,7 +12,7 @@ export interface SPSPService {
   GETPayEndpoint(ctx: AppContext): Promise<void>
 }
 
-interface ServiceDependencies extends BaseService {
+interface ServiceDependencies extends Omit<BaseService, 'knex'> {
```

Comment:
another #131 fwiw:
I'm looking into removing `knex` from `BaseService` and having services access it (or a per test transaction) via [`.knex()`](https://vincit.github.io/objection.js/api/model/static-methods.html#static-knex) on their model